helm-chart/kuberay-operator/values.yaml

```diff
@@ -55,3 +55,12 @@ readinessProbe:
 
 createCustomResource: true
 rbacEnable: true
+
+# Respect the "restricted" security policy
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop: ["ALL"]
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
```

Comment:
Shall we add this to the non-Helm operator config as well?

Reply:
Yyeah, actually let me open an issue about just that, because I've just experienced a massive amount of pain updating a bunch of copy-pasted configuration.

Reply:
But I think we only duplicate Ray operator configuration once (Helm and non-Helm). Maybe we can make an unprincipled exception just this once for the operator itself?

Reply:
https://github.com/ray-project/kuberay/issues/765

Reply:
Resolved